SecurityPkg/SecurityPkg.dec

```diff
@@ -666,5 +666,12 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdEnforceSelfsignedPk|FALSE|BOOLEAN|0x00010024
 # MU_CHANGE [END]
 
+# MU_CHANGE [BEGIN] - 108842
+  ## Indicates if the device should be in "ship mode" or not.
+  # TRUE  - Device is in ship mode and thus TPM should behave normally
+  # FALSE - Device is not in ship mode and TPM should be lax
+  gEfiSecurityPkgTokenSpaceGuid.PcdEnableShipMode|TRUE|BOOLEAN|0x00010028

```

Comment:
Let's scope this down to what it is actually doing, which which seem to be disallowing the persistent clear permission request